### PR TITLE
Feat/server info

### PR DESCRIPTION
**Pull request by locothedev**
TWO COMPONENTS.

Widget + ServerInfo

No tests yet but can be reviewed already

Known issues:
colors of 'revive' - 'update' and action buttons cannot be matched . Button is not cutomizable in text/icon color

### CODE REVIEW

File: src/components/Widget/Widget.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+const getHint = (hint: string) => {
+    return (
+        <HintStyled>
+            <Tooltip text={hint} children={[<Icon fill={colors.blue} size={14} svg={iconTypes.helpCircle}/>]} />
+        </HintStyled>
+    )
+}
+
+const Widget: React.FC<WidgetProps> = ({
+    id,
```
Comment (BillyG83) [inline src/components/Widget/Widget.tsx:18]: please sort alphabetically

File: src/components/Widget/Widget.style.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+export const WidgetTitle = styled.div`
+  color: ${colors.grey};
+  display: flex;
+  justify-content: space-between;
+  font-size: 12px;
+  margin: 0;
+  
+`
+
+export const WidgetInfo = styled.div`
```
Comment (BillyG83) [inline src/components/Widget/Widget.style.ts:40]: this should prob be a styled h3 and use font.heading and fonts.h3 please =]

File: src/components/Widget/Widget.style.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  position: relative;
+  width: 100%;
+
+`
+
+export const WidgetInfoBox = styled.div`
+  margin-right: 10px;
+`
+
+export const WidgetTitle = styled.div`
```
Comment (BillyG83) [inline src/components/Widget/Widget.style.ts:31]: a title that is smaller text size than the Info? are these mixed up maybe? Also please consider to use semantic tags like <p> and <h3> etc, divs are non semantic so it makes it hard to less abled users to understand the apps they are trying to use

File: src/components/Widget/Widget.style.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  align-items: center;
+  border: 1px solid ${colors.paleBlue2};
+  border-radius: 12px;
+  box-shadow: inset 0 0 10px rgba(46, 125, 175, 0.15);
+  display: flex;
+  gap: 5px;
+  padding: 16px;
+  position: relative;
+  width: 100%;
+
```
Comment (BillyG83) [inline src/components/Widget/Widget.style.ts:24]: remove this line please, is the formatting still not working on PC?

File: src/components/Widget/Widget.style.ts [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+export const WidgetStyled =  styled.div`
+  ${resetCSS}
+  ${fonts.text};
+  align-items: center;
+  border: 1px solid ${colors.paleBlue2};
+  border-radius: 12px;
+  box-shadow: inset 0 0 10px rgba(46, 125, 175, 0.15);
+  display: flex;
+  gap: 5px;
```
Comment (BillyG83) [inline src/components/Widget/Widget.style.ts:20]: gap with flex, does this work? i'm learning =]

File: src/components/Widget/Widget.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    description,
+    hint,
+    icon,
+    title
+}: WidgetProps) => {
+    return (
+        <WidgetStyled id={id}>
+            {icon && <div style={{display: "grid", placeItems: "center", marginRight: '10px'}}><Icon size={30} fill={colors.blue} svg={icon}/></div>}
```
Comment (BillyG83) [inline src/components/Widget/Widget.tsx:26]: ok now i see the gird, excuse one of previous comments, please move this to styles file =]

File: src/components/ServerInfo/ServerInfo.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,55 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import React from 'react';
+import ServerInfo from './ServerInfo';
+import { iconTypes } from '../Icon/collection'
+import { Widget } from '../Widget';
+
+
```
Comment (BillyG83) [inline src/components/ServerInfo/ServerInfo.stories.tsx:7]: dam prettier, whys it not work on PC?

Comment (locothedev) [reply src/components/ServerInfo/ServerInfo.stories.tsx:7]: WINDOWS IS SHIT

File: src/components/Widget/Widget.style.ts (hunk 5 above)
Comment (locothedev) [reply src/components/Widget/Widget.style.ts:20]: Im pretty sure it works :D

File: src/components/Widget/Widget.style.ts (hunk 4 above)
Comment (locothedev) [reply src/components/Widget/Widget.style.ts:24]: Bill Gates does not want me to write pretty code :(

File: src/components/Widget/Widget.tsx (hunk 1 above)
Comment (locothedev) [reply src/components/Widget/Widget.tsx:18]: yes sir

File: src/components/Widget/Widget.style.ts (hunk 2 above)
Comment (locothedev) [reply src/components/Widget/Widget.style.ts:40]: gotchyaa

File: src/components/Widget/Widget.style.ts (hunk 3 above)
Comment (locothedev) [reply src/components/Widget/Widget.style.ts:31]: Figma says . Title smaller than Info

**Comment by locothedev** [comment]
This will be merged into admin refactor

**Pull request closed without merging**